chainer/functions/connection/n_step_lstm.py

```diff
@@ -194,49 +237,100 @@ def n_step_bilstm(
+            where ``I`` is size of input. ``ws[i][j]``  where ``2 <= i`` and
+            ``0 <= j < 4`` is ``(N, 2N)`` because they are multiplied with two
+            hidden layers :math:`h_t = [h^{f}_t; h^{b}_t]`. All other matrices
+            has ``(N, N)`` shape.
+        bs (list of list of :class:`~chainer.Variable`): Bias vectors.
+            ``bs[2 * l + m]`` represents weights for l-th layer of m-th
+            direction. (``m == 0`` means the forward direction and ``m == 1``
+            means the backward direction.)
             Each ``bs[i]`` is a list containing eight vectors.
-            ``bs[i][j]`` is corresponding with ``b_j`` in the equation.
-            Shape of each matrix is ``(N,)`` where ``N`` is dimention of
-            hidden units.
-        xs (list of chainer.Variable): A list of :class:`~chainer.Variable`
+            ``bs[i][j]`` is corresponding with :math:`b_j` in the equation.
+            Shape of each matrix is ``(N,)``.
+        xs (list of :class:`~chainer.Variable`):
+            A list of :class:`~chainer.Variable`
             holding input values. Each element ``xs[t]`` holds input value
             for time ``t``. Its shape is ``(B_t, I)``, where ``B_t`` is
-            mini-batch size for time ``t``, and ``I`` is size of input units.
-            Note that this functions supports variable length sequences.
-            When sequneces has different lengths, sort sequences in descending
-            order by length, and transpose the sorted sequence.
-            :func:`~chainer.functions.transpose_sequence` transpose a list
-            of :func:`~chainer.Variable` holding sequence.
+            mini-batch size for time ``t``. Note that this functions supports
+            variable length sequences. When sequneces has different lengths,
+            sort sequences in descending order by length, and transpose the
+            sorted sequence. :func:`~chainer.functions.transpose_sequence`
+            transpose a list of :func:`~chainer.Variable` holding sequence.
             So ``xs`` needs to satisfy
             ``xs[t].shape[0] >= xs[t + 1].shape[0]``.
 
     Returns:
         tuple: This functions returns a tuple concaining three elements,
-            ``hy``, ``cy`` and ``ys``.
+        ``hy``, ``cy`` and ``ys``.
 
             - ``hy`` is an updated hidden states whose shape is same as ``hx``.
             - ``cy`` is an updated cell states whose shape is same as ``cx``.
             - ``ys`` is a list of :class:`~chainer.Variable` . Each element
               ``ys[t]`` holds hidden states of the last layer corresponding
-              to an input ``xs[t]``. Its shape is ``(B_t, N)`` where ``B_t`` is
-              mini-batch size for time ``t``, and ``N`` is size of hidden
+              to an input ``xs[t]``. Its shape is ``(B_t, 2N)`` where ``B_t``
+              is mini-batch size for time ``t``, and ``N`` is size of hidden
```

Comment:
* ... is the mini-batch size
* ... N is the size of the hidden units.

Reply:
I fixed next commit.